### PR TITLE
Task/8 Results will be saved in database

### PR DESCRIPTION
**Pull request by huangs08**

### CODE REVIEW

File: Backend/src/main/config/config.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+//cloud database information
+const DB_DATABASE = "moodipy";
```
Comment (nasaku898) [inline Backend/src/main/config/config.js:2]: put db info in env file

Comment (nasaku898) [reply Backend/src/main/config/config.js:2]: you can download the env file on discord moodipy channel

File: Backend/src/main/dao/SubmissionDAO.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
+const db = require('../helpers/db');
+
+insertResult = (initial_emotion, api_emotion, user_feedback) => {
+
+    function isEmpty(str) {
```
Comment (nasaku898) [inline Backend/src/main/dao/SubmissionDAO.js:5]: use arrow function

File: Backend/src/main/dao/SubmissionDAO.js [hunk 3]
```diff
@@ -2,7 +2,7 @@ const db = require('../helpers/db');
 
 insertResult = (initial_emotion, api_emotion, user_feedback) => {
 
-    function isEmpty(str) {
+    isEmpty = (str) => {
```
Comment (nasaku898) [inline Backend/src/main/dao/SubmissionDAO.js:5]: you should write const isEmpty =

Comment (huangs08) [reply Backend/src/main/dao/SubmissionDAO.js]: done

**Review by nasaku898 — APPROVED** [review]

**Pull request merged**